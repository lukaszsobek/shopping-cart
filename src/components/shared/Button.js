import React from "react";

const Button = (props) => {
    const { action, className, label } = props;
    return (
        <button
            className={ className }
            onClick={ action }>{ label }</button>
    );
}

export default Button;
