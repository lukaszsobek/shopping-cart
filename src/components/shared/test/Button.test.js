import Button from "../Button";

describe("Renders",() => {
    it("button", () => {
        const props = {
            className: "classNameTest",
            action: () => null,
            type: "button"
        }
        expect(Button(props)).toMatchSnapshot();
    })
})