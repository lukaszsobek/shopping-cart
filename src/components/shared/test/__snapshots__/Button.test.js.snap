// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders button 1`] = `
<button
  className="classNameTest"
  onClick={[Function]}
  type="button"
/>
`;
